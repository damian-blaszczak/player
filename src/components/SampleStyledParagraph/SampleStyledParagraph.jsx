import styled from 'styled-components';

const SampleStyledParagraph = styled.p`
  font-size: 1.4rem;
  text-align: center;
  margin-top: 10px;
`;

export default SampleStyledParagraph;
