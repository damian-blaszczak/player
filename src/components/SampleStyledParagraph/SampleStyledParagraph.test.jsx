import React from 'react';
import { render } from '@testing-library/react';
import SampleStyledParagraph from './SampleStyledParagraph';

describe('SampleStyledParagraph', () => {
  it('renders correctly', () => {
    const { container } = render(
      <SampleStyledParagraph />
    );
    expect(container).toMatchSnapshot();
  });

  it('should have font-size equal to 1.4rem', () => {
    const { container } = render(
      <SampleStyledParagraph />
    );

    const paragraphElement = container.querySelector('p');
    expect(paragraphElement).toHaveStyle('font-size: 1.4rem');
  });
});
