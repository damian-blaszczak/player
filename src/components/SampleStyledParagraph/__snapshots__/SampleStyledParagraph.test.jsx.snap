// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleStyledParagraph renders correctly 1`] = `
<div>
  <p
    class="sc-AxjAm DyKgd"
  />
</div>
`;
